### DISCUSSION TITLE
Twitter. Inc algorithms

### DISCUSSION THREAD

**Discussion by FerasK0** [robertodoering/harpy#629] [General]
Hello Robert. 

Apparently Elon Musk bought Twitter, but I'm not here for the poltics, supposedly Elon Musk has intentions to make Twitter's algorithms open source. ([See here](https://twitter.com/elonmusk/status/1518677066325053441?t=kwegByb1X0YLLWmJVNj5cA&s=19))

I want to know what's gonna happen to harpy after the purchase of Twitter.

And what features is excepted to come sooner than before because of this decision?

**Comment by robertodoering** [comment]
I doubt this will change anything for third party Twitter apps. I wouldn't know, though :D
